src/components/Logo/BrandMark.js

```diff
@@ -40,16 +41,19 @@ const styles = {
   })
 }
 
-const R = ({ fill }) => (
-  <div {...styles.container}>
-    <svg {...styles.svg} viewBox={VIEWBOX}>
-      <path fill={fill} d={PATH} />
-    </svg>
-  </div>
-)
-
-R.defaultProps = {
-  fill: '#000'
+const R = ({ fill }) => {
+  const [colorScheme] = useColorContext()
+  return (
+    <div {...styles.container}>
+      <svg
+        {...styles.svg}
+        viewBox={VIEWBOX}
+        {...colorScheme.set('fill', 'logo')}
```

Comment:
why both a svg fill and a path fill? And why is only the svg from props?

Reply:
the path fill allow overwrite to get a specific color, IMO can be kept. Setting the fill attr on the svg would not work because the css is more specific if on the same element

Reply:
one alternative would be:
```suggestion
        {...colorScheme.set('fill', fill || 'logo')}
```
an then remove the fill on the path.